sampleBrokers.md

```diff
@@ -0,0 +1,46 @@
+# Sample Service Brokers
+
+The following example service broker implementations have been developed
+as a starting point if you are developing your own service broker.
+
+The Open Service Broker API does not make any statement as to the
+validity, stability or compliance of any of them.
+
+If you would like to add additional brokers to this list open a
+a pull request against 
+[this repository](https://github.com/openservicebrokerapi/servicebroker).
```

Comment:
Can you please add more comments for what is the content of the PR? [this repository](https://github.com/openservicebrokerapi/servicebroker) is just the API specification.